drivers/storage/portworx/preflight.go

```diff
@@ -44,6 +44,8 @@ const (
 	DefCmetaAzure = "type=Premium_LRS,size=64"
 	// DefCmetaGKE default metadata cloud device for DMthin GKE
 	DefCmetaGKE = "type=pd-ssd,size=64"
+	// preFlightOutputLog log location for pre-flight output
+	preFlightOutputLog = "/var/cores/px-pre-flight-output.log"
```

Comment:
nit: why not shorten it to "px-pre-flight.log"? "output" is implied, right?

Reply:
Well I can't use that log file name cause already being used to save the pre-flight results.   So rather than change that location I just added the extension.     
```
-rw------- 0/0 94250 2023-11-11 02:14 /var/cores/px-pre-flight-output.log
-rw-r--r-- 0/0 621 2023-11-11 02:14 /var/cores/px-pre-flight.log
```